README.md

```diff
@@ -183,6 +183,7 @@ interface ShortUrlOptions {
   domain?: string;
   // Length of the code
   shortCodeLength?: number;
+  // Tells if shlink should validate a status 20x is returned (after following redirects) when trying to shorten a URL
```

Comment:
I had a lot of trouble reading that

I think `Tells if shlink should validate that a 20x status is returned (after following redirects) when trying to shorten a URL` is clearer, what do you think ?